temporal-normalizer/tools/TimeML-Normalizer/README.TXT

```diff
@@ -0,0 +1,41 @@
+====
```

Comment:
Do we want to keep everything in `temporal-normalizer/tools/TimeML-Normalizer`? We seem to be using it as a maven dependency; right?